linkerd/concurrency-limit/src/lib.rs

```diff
@@ -69,39 +62,30 @@ where
     type Future = ResponseFuture<S::Future>;
 
     fn poll_ready(&mut self, cx: &mut Context<'_>) -> Poll<Result<(), Self::Error>> {
-        trace!(available = %self.semaphore.available_permits(), "acquiring permit");
+        trace!(
+            // available = %self.semaphore.available_permits(),
```

Comment:
logging the number of available permits here depends on upstream PR tokio-rs/tokio#3683